pick.c

```diff
@@ -667,19 +667,23 @@ tty_restore(void)
 }
 
 void
-print_line(const char *string, size_t length, int so, ssize_t ulso, ssize_t uleo)
+print_line(const char *str, size_t len, int so, ssize_t ulso, ssize_t uleo)
```

Comment:
I, for one, am of the view to write out as many acronyms as possible. But since we have `ssize_t` unt so weiter already, I'd say this falls in line with that style better.